### DISCUSSION TITLE
Slow response of "dispatching" sensor - OE issue or integration issue?

### DISCUSSION THREAD

**Discussion by DragonQ** [BottlecapDave/HomeAssistant-OctopusEnergy#1021] [General]
I've been using this integration for a year or two and last winter I remember the "dispatching" sensor updating pretty quickly, i.e. within a minute of a slot starting. I rarely used smart charging over the summer but have started again now solar output is lower, and this sensor seems to be taking its time updating. I plugged in my car at 10:01 today and the sensor didn't update until 10:08. Obviously that's 7 minutes where my house battery is discharging instead of charging - not ideal.

Is this due to an integration update frequency change or is it an Octopus Energy issue?

**Comment by liquidfusionv** [comment]
Not the answer you are looking for but might be better to integrate your Charger or EV to HA and have that be the sensor that you rely on instead of Octopus's "dispatching" sensor

**Comment by DragonQ** [comment]
I actually started using the dispatching sensor in the first place because the Zapp has a random multi-minute delay before it starts charging (can't be changed apparently), plus the sensor doesn't update that frequently.

**Comment by BottlecapDave** [comment]
The settings and dispatches were updated every 5 minutes, but as of v13 this has now been reduced to every 3 minutes. The sensor is powered by these dispatches so there will always be a delay in them updating (the API is rate limited), and could be delayed further if OE are slow to updating their dispatch schedule. If the dispatches are already available, then the sensor should update within the minute of the dispatch becoming available.
